README.md

```diff
@@ -1,14 +1,15 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_get_data_DOM/)
+   - [DEMO LINK](https://beliyandr.github.io/js_get_data_DOM/)
```

Comment:
Add this demo link to the PR description 
<img width="970" alt="image" src="https://github.com/mate-academy/js_get_data_DOM/assets/64750958/6e8b8474-2a39-411b-b666-66e619443f16">